cmd/local/options.go

```diff
@@ -22,15 +22,22 @@ type Options struct {
 	version        string
 	profile        string
 	components     []string
+	values         []string
 }
 
-func defaultComponentList() []keb.Components {
-	defaultComponents := []string{"cluster-essentials", "istio-configuration@istio-system",
+func defaultComponents() []string {
+	return []string{"cluster-essentials", "istio-configuration@istio-system",
 		"certificates@istio-system", "logging", "tracing", "kiali", "monitoring", "eventing", "ory", "api-gateway",
```

Comment:
Can'w we generate this list from this URL: https://github.com/kyma-project/kyma/blob/main/installation/resources/components.yaml ??

The list of existing components should come from Kyma and not be defined in the reconciler.

Reply:
Code for reading the component list is available here: https://github.com/kyma-incubator/reconciler/pull/123/files#diff-ca4a1604ae2c727c0927d025dc68f9e56e274b5dbbd55ad6d66c45bebaf15bc8

Reply:
Fixed. Default is taken from workspace. You can override it with your own file using `--components-file` option.